src/NUnitFramework/mock-assembly/mock-assembly-netstandard13.csproj

```diff
@@ -63,14 +64,17 @@
     <None Include="..\..\nunit.snk">
       <Link>nunit.snk</Link>
     </None>
-    <None Include="mock-assembly-portable.project.json" />
+    <None Include="mock-assembly-netstandard13.project.json" />
   </ItemGroup>
   <Import Project="$(MSBuildExtensionsPath32)\Microsoft\Portable\$(TargetFrameworkVersion)\Microsoft.Portable.CSharp.targets" />
```

Comment:
I tried removing them just to be sure, but they are needed. Technically, these are portable projects, that is why you see the (Portable) after them in solution explorer.